signac/__main__.py

```diff
@@ -361,6 +361,7 @@ def main_schema(args):
 
 def main_sync(args):
     """Handle sync subcommand."""
+    # TODO: This function appears to be untested.
     #
     # Valid provided argument combinations
```

Comment:
Unrelated to this PR, but I think this is supposed to say:
```suggestion
    # Validate provided argument combinations
```